### DISCUSSION TITLE
Make argon pro hitobjects translucent

### DISCUSSION THREAD

**Discussion by AtakanErmis** [ppy/osu#21832] [Ideas]
in most of the skins that players used to, hitobjects are actually translucent. This improves readability for especially stacked/overlapping objects.

for comparison (current "argon" pro vs BTMC Freedom DiVE skin):

![osu_2022-12-27_23-13-38](https://user-images.githubusercontent.com/16894959/209717999-0ea4b7be-2755-45e5-ae6c-6adf4a7cf11d.jpg)
![osu_2022-12-27_23-13-48](https://user-images.githubusercontent.com/16894959/209718015-8914693a-2bf4-4f26-8c0b-b5d5b06756cf.jpg)

**Comment by DutchEllie** [comment]
Y'know, looking at the still image of the Argon skin, I think that is actually a huge contributor to why I find it so hard to use. It's hard to notice exactly what's causing it when the circles are flying by, but not being able to see the circles under the slider is probably a huge factor!

Maybe I've gotten used to it too, but the slider not being transparent looks kinda sloppy too, especially when the blue circles that are fading out look so stunning

**Comment by deanveloper** [comment]
I came here to say the exact same thing. The opaque-ness of the skin makes it especially difficult to see hitcircles which appear directly after sliders, as well as differentiating between doubles and triples.
